### PR TITLE
timeouts per suite

### PR DESCRIPTION
**Pull request by diasdavid**

### CODE REVIEW

**Comment by Dignifiedquire** [comment]
Why not a global config in `test/setup.js` or directly passed to mocha?

File: test/api/add.spec.js [hunk 1]
```diff
@@ -21,13 +21,13 @@ if (isNode) {
 }
 
 describe('.add', function () {
+  this.timeout(60000)
+
   it('add file', function (done) {
```
Comment (Dignifiedquire) [inline test/api/add.spec.js]: Please move these `function`s to `=>` functions as the only reason I kept the inconsistency was the need for calling `this.timeout`

Comment (diasdavid) [inline test/api/add.spec.js]: ok, sounds good

**Comment by diasdavid** [comment]
I took out the local timeouts (I went through it because when I went through docs, it was only specified test and suit level timeouts https://mochajs.org/#more-information)

How can I set timeouts in karma?

**Comment by diasdavid** [comment]
Now only missing:

- [ ] remove `function` key word and make @Dignifiedquire happy :)

**Comment by Dignifiedquire** [comment]
mocha-node pass it to the mocha call in the guplfile as an option https://github.com/sindresorhus/gulp-mocha/blob/master/readme.md

karma: in the karma conf set client.mocha.timeout https://github.com/karma-runner/karma-mocha/blob/master/README.md

File: karma.conf.js [hunk 2]
```diff
@@ -46,6 +46,11 @@ module.exports = function (config) {
   config.set({
     basePath: '',
     frameworks: ['mocha'],
+    client: {
+      mocha: {
+        timeout: 60000
```
Comment (Dignifiedquire) [inline karma.conf.js]: Let's pull this into `tasks/config.js` so that it's only defined in one place

File: tasks/config.js [hunk 3]
```diff
@@ -42,7 +42,8 @@ const shared = {
     tls: '{}',
     console: '{}',
     'require-dir': '{}'
-  }
+  },
+  timeout: 60000
```
Comment (Dignifiedquire) [inline tasks/config.js]: This should not be namespaced under `webpack`, as it is not a webpack relevant config. Better would be `test = {timeout: 60000}`

**Pull request merged**